website/docs/getting-started.mdx

```diff
@@ -22,7 +22,7 @@ yarn add victory-native
 Now you should be ready to go.
 
 :::info
-> `victory-native-xl` diverged from the original `victory-native` after version 36+. If you encounter issues, please force the installed version with `yarn add victory-native@40.0.3`.
+> `victory-native-xl` diverged from the original `victory-native` after version 36+. If you encounter issues, please force the installed version with `yarn add victory-native`.
```

Comment:
Do we even need this `:::info` block at all anymore?

Reply:
the blue is so pretty, but yeah, i realized that too once I previewed, removed!